### DISCUSSION TITLE
Too many programs, how to scroll?

### DISCUSSION THREAD

**Discussion by kydar1** [Open-Shell/Open-Shell-Menu#1346] [General]
I use the Classic style start menu.  When I hover over Programs, 4 filled columns of programs/apps appear.  But I have more than that installed - the rest are off the screen.  Is there a setting to scroll start menu programs that are too numerous to appear on screen? (Note: I know that the Windows 7 style start menu would "fix" it, but I am not interested in switching to that.  Also don't want to change screen resolution to make more columns fit on screen.  Thanks.

**Comment by tallman69** [comment]
On the General Behavior tab of the Settings for Open-Shell, under **Multiple columns in sub-menus** select **Use single column (scroll)**.  That should do it.

**Reply by kydar1** [reply]
Dang it, and I looked thru the settings before I posted.  Thank you!  In fact, I found that just unchecking "All Columns are same size" caused a fifth column to appear and that works even better for me.  Thanks again.

**Reply by tallman69** [reply]
You're welcome!  Glad you found an even better fix.
